client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx

```diff
@@ -22,6 +27,12 @@ function NewRideInfo() {
   const specialRequestsDefaultValue = DRIVER_CAPABILITIES.map(({ value }) => value);
   const selectedSpecialRequests = watch('specialRequest', specialRequestsDefaultValue) || [];
 
+  const [serviceType, setServiceType] = React.useState('Ride');
```

Comment:
@aviv-taori Should use the form ride data, no need for a separate state

Reply:
Needed for default. No?

Reply:
@aviv-taori You can set the default in the `useForm`

Reply:
Ok!